src/modules/markets/MarketAssetsListItem.tsx

```diff
@@ -68,18 +70,42 @@ export const MarketAssetsListItem = ({ ...reserve }: ComputedReserveData) => {
       </ListColumn>
 
       <ListColumn>
-        <FormattedNumber compact value={reserve.totalLiquidity} variant="main16" />
-        <ReserveSubheader value={reserve.totalLiquidityUSD} />
+        {reserve.symbol === GHO_SYMBOL ? (
+          '--'
```

Comment:
Instead of rendering `--` here you can use this component:

```
<NoData variant="secondary14" color="text.secondary" />
```

That way it'll match the other columns with no data

Reply:
Removed gho stuff in this PR